code/modules/cargo/bounties/engineering.dm

```diff
@@ -24,6 +24,19 @@
 	description = "Station 49 is looking to kickstart their research program. Ship them a tank full of Tritium."
 	gas_type = /datum/gas/tritium
 
+/datum/bounty/item/engineering/pacman
+	name = "P.A.C.M.A.N.-type portable generator"
+	description = "A sister station had a problem with the SME, and now need something to power there console to call their shuttle. Can you send them a pacman to fix this for us?"
+	reward = 3500 //2500 for the cargo one
```

Comment:
Isn't it bad if you get more as a reward than it costs to order a new one?

Reply:
potted plants are like this as well, And its only a net of 1400 if they buy it, well if they make or get a full one they get the full 3500